src/MDFTextAccessibility.m

```diff
@@ -155,10 +155,31 @@ + (BOOL)textColor:(nonnull UIColor *)textColor
 }
 
 + (BOOL)isLargeForContrastRatios:(nullable UIFont *)font {
+  if (font.pointSize >= 18) {
+    return YES;
+  }
+  if (font.pointSize < 14) {
+    return NO;
+  }
+
   UIFontDescriptor *fontDescriptor = font.fontDescriptor;
-  BOOL isBold =
-      (fontDescriptor.symbolicTraits & UIFontDescriptorTraitBold) == UIFontDescriptorTraitBold;
-  return font.pointSize >= 18 || (isBold && font.pointSize >= 14);
+  if ((fontDescriptor.symbolicTraits & UIFontDescriptorTraitBold) == UIFontDescriptorTraitBold) {
+    return YES;
+  }
+
+  // We treat medium as large for accesibility when larger than 14.
+  NSDictionary *fontTraits = [fontDescriptor objectForKey:UIFontDescriptorTraitsAttribute];
+  NSNumber *fontWeight = fontTraits[UIFontWeightTrait];
+#if defined(__IPHONE_8_2)
+  CGFloat fontWeightMedium = UIFontWeightMedium;
+#else
+  CGFloat fontWeightMedium = 0x3fcd70a3e0000000; // Value gotten from "po UIFontWeightMedium"
```

Comment:
Err. If this is legit, can we link to a SO post or some other discussion? Also, this is probably a totally normal floating point number if you don't cast it to a hex value? What value is that?